aten/src/ATen/native/sparse/cuda/SparseBlasImpl.cpp

```diff
@@ -795,6 +795,56 @@ void addmm_out_sparse_csr(
   }
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~linalg_solve~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+template <typename scalar_t>
+void _apply_sparse_csr_lu_solve(
```

Comment:
I want to keep this file only for cuSPARSE based functions. Could you please move the implementation to some other file? It's not easy to add a new file, so using `SparseCsrTensorMath.cu` would be fine.

Reply:
Yep, sure! Done. :)